conda/cli/main_mock_activate.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Mock CLI implementation for `conda activate`.
+
+A mock implementation of the activate shell command for better UX.
+"""
+
+
+def execute(args, parser):
+    print("ERROR: Run 'conda init' before 'conda activate'")
+    return 1
```

Comment:
```suggestion
from .. import CondaError


def execute(args, parser):
    raise CondaError("Run 'conda init' before 'conda activate'")
```

Let's raise a `CondaError` here instead. This has the following benefits:

- It's the standard way of showing errors to our users
- It automatically handles returning 1 as the exit code